api/lib/ZwaveClient.ts

```diff
@@ -2184,18 +2181,15 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 			},
 			logConfig: {
 				// https://zwave-js.github.io/node-zwave-js/#/api/driver?id=logconfig
-				enabled: this.cfg.logEnabled,
-				level: this.cfg.logLevel
-					? loglevels[this.cfg.logLevel]
-					: 'info',
-				logToFile: this.cfg.logToFile,
+				...utils.buildLogConfig({
+					logEnabled: this.cfg.logEnabled,
+					logLevel: this.cfg.logLevel,
+					logToFile: this.cfg.logToFile,
+					maxFiles: this.cfg.maxFiles,
+					nodeFilter: this.cfg.nodeFilter,
+				}),
 				filename: ZWAVEJS_LOG_FILE,
 				forceConsole: isDocker() ? !this.cfg.logToFile : false,
```

Comment:
forceConsole depends from logToFile so that should be included as well in buildLogConfig and also filename